docs/learn/universal-profile/universal-receiver/accept-reject-assets.md

```diff
@@ -94,6 +115,15 @@ Please make sure to unlock MetaMask and disable Browser Extension while doing th
 
 ![Compiling contract in Remix](/img/guides/lsp1/remix-compiling-contract.jpeg)
 
+:::info Deploy using Universal Profiles
```

Comment:
Since this content is also in the _"Working with Remix"_ page, maybe we could just have the callout that simply link to this page instead of duplicating the content?

<img width="1530" alt="image" src="https://github.com/user-attachments/assets/5ebdce08-8b62-4644-8876-414af3a82515" />